installinstallmacos.py

```diff
@@ -506,6 +506,16 @@ def main():
         sys.exit('This command requires root (to install packages), so please '
                  'run again with sudo or as root.')
 
+    current_dir = os.getcwd()
+    if os.path.expanduser("~") in current_dir:
+        bad_dirs = ['Documents', 'Desktop', 'Downloads', 'Library']
+        for bad_dir in bad_dirs:
+            if bad_dir in current_dir:
```

Comment:
Safer to do `if bad_dir in os.path.split(current_dir): ` so you don't get false matches against `/Users/foo/MovedFromDesktop`

Reply:
```
>>> this_dir = "/Users/foo/MovedFromDesktop"
>>> "Desktop" in this_dir
True
>>> "Desktop" in os.path.split(this_dir)
False
```